src/assets/sass/components/_army_card.scss

```diff
@@ -0,0 +1,4 @@
+.material-image {
+  height: 30px;
+  width: 30px;
+}
```

Comment:
I suppose it should be .army-image class, as .material-image is used elsewhere and is exactly the same